kaggle_tools_update.Dockerfile

```diff
@@ -18,8 +18,8 @@ FROM gcr.io/kaggle-images/python:latest
 
 RUN pip install --upgrade git+https://github.com/Kaggle/learntools
 
-# Set up pip to enable pip install.
-ADD patches/kaggle_bashrc /root
-# Patch the system-wide bashrc file for non-root users.
-RUN cat /root/kaggle_bashrc >> /etc/bash.bashrc
-RUN rm /root/kaggle_bashrc
+# Set up an initialization script to be executed before any other commands initiated by the user.
```

Comment:
We shouldn't need this since this inherits this setup from the base image.   Though, given the need of this whole thing (work around a currently broken base image build), it probably makes sense to do here as well.  Could you still document why we have this duplication in an inline comment here?